api/audit-trail-gw/src/services/configuration-service.ts

```diff
@@ -36,7 +36,7 @@ export class ConfigurationService {
 	};
 
 	config: Config = {
-		port: process.env.PORT ? Number.parseInt(process.env.PORT, 10) : 3000,
+		port: process.env.PORT ? Number.parseInt(process.env.PORT, 10) : 3002,
```

Comment:
Default must be always the same in all services (so 3000).
Port mapping must be done outside, either in env variable (if you run with npm run) or via docker-compose mapping or via kubernetes services.
There is no need to have two different ports for the two services.

Reply:
👍